devel/nlohmann-json/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+PortGroup                       cxx11 1.1
+
+github.setup                    nlohmann json 3.2.0 v
+name                            nlohmann-json
+categories                      devel
+platforms                       darwin
+supported_archs                 noarch
+license                         MIT
+maintainers                     {gmail.com:g.litenstein @Lord-Kamina} openmaintainer
+description                     Nlohmann JSON for modern C++.
+long_description                Nlohmann is a header-only (and optionally single-header) JSON library \
+                                written in vanilla C++11. Has Unicode support\; DOM and SAX parsers, among other features.
+checksums                       rmd160  3c04b206efdeaacf49f374b33eec5b86978eef6d \
+                                sha256  2131e11d0774005e68432c5c01ff0a595e6abf0ba7979738a2017ee8a69d4001 \
+                                size    114561095
+
+configure.pre_args-append       -DBUILD_TESTING=OFF -DJSON_BuildTests=OFF
```

Comment:
can just be configure.args-append

Reply:
Can just be `configure.args`. `*.args` is intended to default to empty and to be set by the portfile. `*.pre_args` and `*.post_args` might already be set by base or portgroups.

Reply:
I wrote it the way I did because for some reason configure.args wasn't working as I expected it to. Maybe I was using them wrong. I can certainly try fixing that when I get home.